.circleci/config.yml

```diff
@@ -855,7 +855,7 @@ step-save-out-cache: &step-save-out-cache
   save_cache:
     paths:
       - ./src/out/Default
-    key: v8-out-cache-{{ checksum "src/electron/.depshash" }}-{{ checksum "src/electron/.depshash-target" }}
+    key: v9-out-cache-{{ checksum "src/electron/.depshash" }}-{{ checksum "src/electron/.depshash-target" }}
```

Comment:
@nornagon GN's framework building logic sucks and doesn't clean up things in the `.framework` folder that shouldn't be there.  E.g. The old `ReactiveCocoa.framework` which caused zip manifest failures

Reply:
do these things actually need to be frameworks or can we statically link them?

Reply:
I want to avoid drastic shifts in this PR and maintain the status quo.  Happy to investigate building these things inline in the future though, their are issues with for instance hard coded path expectations and the ShipIt helper binary.  But it might be possible